presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -828,4 +828,70 @@ private static Double mapL2Norm(Block map)
 
         return Math.sqrt(norm);
     }
+
+    @ScalarFunction
+    @Description("binomial confidence interval lower bound using Wilson score")
+    @Nullable
+    @SqlType("double")
+    public static Double wilsonIntervalLower(@SqlType(StandardTypes.DOUBLE) double pos, @SqlType(StandardTypes.DOUBLE) double neg, @SqlType(StandardTypes.DOUBLE) double z)
+    {
+        if (pos < 0 || neg < 0 || z < 0) {
+            return null;
+        }
+
+        if (pos + neg == 0) {
+            return 0.0;
+        }
+
+        double n = pos + neg;
+        double pHat = pos / n;
+
+        double mean = pHat + z * z / (2 * n);
```

Comment:
Add explicit parenthesis to all these to make the expected order of operations clear for operations that have different precedence (e.g division and multiplication)